### PR TITLE
address review comments > LOBSTER citation, similar packages

### LINKED ISSUE #225: Suggestions for JOSS paper
Here are some suggestions for the JOSS paper:

- citation for LOBSTER
- are there any tools similar to lobsterpy in the community?

### PR DESCRIPTION
**Pull request by naik-aakash**
Closes #225 

# Changes

1. Added LOBSTER citations
2. Added a section of similar and related packages

### CODE REVIEW

**Comment by naik-aakash** [comment]
@JaGeo , this PR could be merged if you think this additions are fine.

**Comment by JaGeo** [comment]
But the reviewer should decide if it closes his remarks. I will merge after he confirms. Will read it soon

File: paper/paper.md
Comment (JaGeo) [inline paper/paper.md:73]:
```suggestion
.
```

File: paper/paper.md
Comment (JaGeo) [inline paper/paper.md:74]:
```suggestion
Other software packages that enable visualizing results specifically from the LOBSTER software are wxDragon[@wxdragon] and [@abipy].
```

File: paper/paper.md
Comment (JaGeo) [inline paper/paper.md:75]:
```suggestion
LobsterPy differs from these two packages by providing further analysis of the calculations, interpretable text summaries and featurizers for ML studies besides plotting the data.
```

**Review by JaGeo — CHANGES_REQUESTED** [review]
I left some ideas. Please merge or refine further, @naik-aakash

**Comment by naik-aakash** [comment]
> I left some ideas. Please merge or refine further, @naik-aakash

Done 😃

File: paper/paper.md
Comment (JaGeo) [inline paper/paper.md:72]:
```suggestion
LobsterPy can be seen to be similar in spirit to sumo [@Ganose2018], as both provide Python tools to analyze and visualize data related to the electronic structure that are based on ab initio calculations. Other software packages that enable visualizing results specifically from the LOBSTER software are wxDragon[@wxdragon] and [@abipy]. LobsterPy differs from these two packages by providing further analysis of the calculations, interpretable text summaries, and featurizers for ML studies besides plotting the data.
```

**Review by JaGeo — APPROVED** [review]

**Pull request merged**
